tests/test_earthspy.py

```diff
@@ -330,6 +330,20 @@ def test_sentinelhub_request(self) -> None:
         # # check that a Sentinel Hub request was created
         assert isinstance(sr2, shb.SentinelHubRequest)
 
+    def test_geojson(self) -> None:
+        """Test geojson files"""
+
+        for file in os.listdir("earthspy/data"):
+            with open(f"earthspy/data/{file}", "r+") as files_geo:
+                data = json.load(files_geo)
+                array_coord = np.array(data["features"][0]["geometry"]
+                                           ['coordinates'][0])
+                
+                assert data["features"][0]["geometry"]['type'] == "Polygon"
+                assert array_coord.shape == (5, 2)
+                assert ((array_coord >= -90) & (array_coord <= 90)).all()
+                assert array_coord[0, 0] == array_coord[-1, 0]
```

Comment:
please add a comment before each of the test explaining what you're doing, just like in the rest of earthspy tests.